guides/v2.1/cloud/env/variables-global.md

```diff
@@ -0,0 +1,72 @@
+---
+group: cloud-guide
+title: Global variables
+functional_areas:
+  - Cloud
+  - Configuration
+---
+
+The following _global_ variables control actions in the build, deploy, and post-deploy stages of the `.magento.env.yaml` file. Because global variables impact every stage, you must set them in the `global` stage. Insert these variables in the `global` stage of the `.magento.env.yaml` file:
+
+```yaml
+stage:
+  global:
+    GLOBAL_VARIABLE_NAME: value
+```
+### `MIN_LOGGING_LEVEL`
+
+-  **Default**—_Not set_
+-  **Version**—Magento 2.1.4 and later
+
+Overrides the minimum logging level for all output streams without making changes to the code. This helps to improve troubleshooting problems with deployment. For example, if your deployment fails, you can use this variable to increase the logging granularity globally. See [Set up notifications—Log levels]({{ page.baseurl }}/cloud/env/setup-notifications.html#log-levels). The `min_level` value in Logging handlers overwrites this setting.
+
+```yaml
+stage:
+    global:
+        MIN_LOGGING_LEVEL: debug
+```
+### `SCD_ON_DEMAND`
+
+-  **Default**—_Not set_
+-  **Version**—Magento 2.1.4 and later
+
+Enable generation of static content when requested by a user. This is ideal for the development and testing workflow, because it decreases the deployment time.
```

Comment:
This is ideal ...  seems vague --   Maybe:  During development and testing, you can set this variable to true to decrease deployment time.      --  it makes me wonder about what would happen if I left it set in production workflow.